var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -248,8 +248,13 @@ def configure_args(self):
             static_bootstrap_flags = '-static-libstdc++ -static-libgcc'
             binutils_options = [
                 '--with-sysroot=/',
-                '--with-stage1-ldflags={0} {1}'.format(
-                    self.rpath_args, static_bootstrap_flags),
+                # The following line causes linkage error on RHEL7
+                # static libstdc++ not being available by default.
+                # For stage1, we just want to produce a working c++ compiler,
+                # we can still rely on host infrastructure to do so.
+                # '--with-stage1-ldflags={0} {1}'.format(
```

Comment:
Hi @cedricchevalier19, could you remove the commented-out lines and update the comment:

> Don't set -static-libstdc++/-static-libgcc for --with-stage1-ldflags since a static libstdc++ is not available on all systems (RHEL7 for example). [THE REST IS COPIED] For stage1, we just want to produce a working c++ compiler, we can still rely on host infrastructure to do so.

Or however you want to rephrase it to make it clear without the commented-out code.

Thanks for the PR!